content/en/calendar/_index.md

```diff
@@ -8,14 +8,15 @@ draft: false
 <section class="section">
   <div class="container">
     <div class="row align-items-center">
-        <p>Welcome to the InnerSource Commons Calendar! Our events and working groups are open to everyone because we know that magic happens when people come together to share their experience and knowledge. 
-        <p>A note about this calendar ... If you're using google, our calendar may display in the GMT timezone. To make sure you see events in your own timezone, please open the calendar on an incognito browser window or use a different search engine.
```

Comment:
Does the calendar exist in other languages and those pages need the note removed as well?